data/FilterList.json

```diff
@@ -3574,12 +3574,12 @@
     "publishedDate": null,
     "submissionUrl": null,
     "syntaxId": null,
-    "viewUrl": "http://rlwpx.free.fr/WPFF/hrsk.7z"
+    "viewUrl": "https://v.firebog.net/hosts/Airelle-hrsk.txt"
```

Comment:
nice! note to self: https://github.com/collinbarrett/FilterLists/issues/201#issuecomment-415001371

Reply:
...too bad there isn't one for id 185. I don't seem to see it.